js/app.js

```diff
@@ -0,0 +1,67 @@
+//Section 1: Terms
+// 1. concatenation is combining two strings together. An example would be if you have two variables that you combine in a console.log as shown below:
+const pet = "dog";
+const name = "Mace";
+console.log("My " + pet + "'s " + "name is " + name);
+// 2. DRY means Don't Repeat Yourself. This aims to reduce repitition. Writing Loops for example can assist in writing DRY code.
+
+//Section 2
+const a = 4;
+const b = 53;
+const c = 57;
+const d = 16;
+const e = 'Kevin';
+
+console.log(a < b);
+console.log(c > d);
+console.log('Name' === 'Name');
+console.log(a > b < c);
+console.log(a === a < d);
```

Comment:
did you test this out?  this prints false.  and also doesn't make sense as an expression.
this should work:
```
a <= a <= d
```